spec/lib/search_spec.rb

```diff
@@ -2221,6 +2223,36 @@ def indexed_post(*args)
       expect(results.more_full_page_results).to eq(nil)
     end
 
+    it 'returns more results flag' do
```

Comment:
Is there a difference between this test and the test at line 2202 as they both share the same description.

Reply:
O this is testing the `search_type: :header`, I guess we need to differentiate between the two tests.

Reply:
Yes, good catch, renamed this test.